src/main/res/values/strings.xml

```diff
@@ -388,13 +388,13 @@
 
     <string name="notification_comment">%s replied to you</string>
     <string name="notification_message">%s sent you a message</string>
-    <string name="notification_message_multiple">New messages on reddit</string>
+    <string name="notification_message_multiple">New messages on Reddit</string>
```

Comment:
I've been informed by reddit staff that the word "reddit" should always be lowercase. If this has changed, it would be good to provide an official source. (applies to all changes of this type)